recipes/jerryscript/all/conanfile.py

```diff
@@ -14,15 +16,82 @@ class JerryScriptStackConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_cmdline": [True, False],
```

Comment:
We usually don't add `build_` prefixes to options.
They only have meaning when building the jerry package, but not when using it.

How about naming these `tool_cmdline`?